tests/shutdown/cleanup_before_shutdown.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Cleanup the system before shutdown
```

Comment:
Seems like currently the summary only restates the filename. Can you mention what the benefit and goal of this module is?

Reply:
Done.